app/src/main/java/dto/Route.java

```diff
@@ -0,0 +1,60 @@
+package dto;
+import java.io.Serializable;
+
+public class Route implements Serializable {
+
+    private String id;
+    private String short_name;
+    private String long_name;
+    private String last_modified_date;
+    private String agency_id;
```

Comment:
I can image you used these variable names to be the same as the json, but the pattern is use camel case to attributes.